src/python/bot/tasks/fuzz_task.py

```diff
@@ -278,8 +277,12 @@ def __init__(self, crashes, context):
       assert crashes[0].security_flag == c.security_flag
 
     self.crashes = crashes
+    fully_qualified_fuzzer_name = context.fuzzer_name
+    if context.fuzz_target:
```

Comment:
nit: why not

```
else:
  fully_qualified_fuzzer_name = context.fuzzer_name
```

IMO that'd be a bit more readable, even though one line more :)

Reply:
Done.